packages/teuchos/core/src/CMakeLists.txt

```diff
@@ -26,9 +26,8 @@ IF (${PROJECT_NAME}_ENABLE_${PARENT_PACKAGE_NAME}ParameterList)
   SET(HAVE_TEUCHOS_PARAMETERLIST TRUE)
 ENDIF()
 
-# Have C++11?
-ASSERT_DEFINED(${PROJECT_NAME}_ENABLE_CXX11)
-SET(HAVE_TEUCHOSCORE_CXX11 ${${PROJECT_NAME}_ENABLE_CXX11})
+# FIXME: Remove this hard-coded setting.  We always have C++11 now.
+SET(HAVE_TEUCHOSCORE_CXX11 ON)
```

Comment:
Right, we should remove this and eliminate the need for a macro of this name in the configured header file.